### PR TITLE
Slack

### PR DESCRIPTION
**Pull request by ManishaBayya**
Please review the code

### CODE REVIEW

File: slack_dump/slack_dump.py [hunk 1]
```diff
@@ -29,28 +29,26 @@ def get_real_time_message(token, start1):
...
 
             if not len(msg[0]) == 1:
                 try:
                     key = hashlib.sha1(msg[0]['channel'] + msg[0]['ts']).hexdigest()
 		    msg[0]["key"] = key
 		    res=db.slack_db.insert_one(msg[0])
 		    print "Inserted!!"
-
-		except Exception:
+		            except Exception:
```
Comment (shivam05011996) [inline slack_dump/slack_dump.py]: I think there an indentation problem here. Could check if the script is running.

File: setup.py [hunk 2]
```diff
@@ -1,7 +1,5 @@
-from setuptools import setup, find_packages
-
 import os
-
+from setuptools import setup, find_packages
 version = '0.1.0'
```
Comment (shivam-deepcompute) [inline setup.py]: we should always separate out import module from starting of the program, please a space.

File: setup.py [hunk 3]
```diff
@@ -1,7 +1,5 @@
-from setuptools import setup, find_packages
-
 import os
-
+from setuptools import setup, find_packages
```
Comment (shivam05011996) [inline setup.py]: we should always separate out import module from starting of the program, please a space.

**Comment by UmangThapliyal** [comment]
please review the code

File: slack_dump/slack_history.py [hunk 4]
```diff
@@ -8,44 +9,91 @@
 db = client.slack_db
 collection = db.slack_db
 
-def get_all_channels_history(auth_token, start_time, end_time):
+
```
Comment (shivam05011996) [inline slack_dump/slack_history.py]: one empty line is enough, to separate them out.

**Review by shivam05011996 — COMMENTED** [review]
Please check for more than 1 extra lines.

**Review by shivam05011996 — APPROVED** [review]

**Pull request closed without merging**